### PR TITLE
General, Medical and Security Records + Exploitable Information for Antags

### PR DESCRIPTION
**Pull request by doshMobile**
## About The Pull Request

Adds records to flesh out characters and make rp gud. Information can be accessed through Medical and Security consoles. Exploitable Information is accessible to antags by clicking on a character. This information can make syndie stuff more than running and gunning and can give some blackmail material to not nice characters.

Code shit itself last minute so it might not be entirely moduled efficiently. All code changed in main files is marked with NON-MODULE CHANGES.

## Why It's Good For The Game

https://www.youtube.com/watch?v=0FA3uTSqasU

## Changelog
:cl:
add: Character Records
add: Exploitable Info
add: One (1) Hairstyle
/:cl:

### CODE REVIEW

**Comment by MrMelbert** [comment]
reee newlines!

File: tgstation.dme [hunk 1]
```diff
@@ -2291,6 +2291,7 @@
 #include "code\modules\mob\mob_update_icons.dm"
 #include "code\modules\mob\status_procs.dm"
 #include "code\modules\mob\transform_procs.dm"
+#include "code\modules\mob\typing_indicator.dm"
```
Comment (MrMelbert) [inline tgstation.dme:2294]: ?

**Review by MrMelbert — COMMENTED** [review]
modularizing looks fine, though a few of the non-module changes could be separated out into new files safely instead

this is a lotta non-module changes so expect merge conflicts in the future

configure whatever text editor you are using to automatically add newlines so the linter stops getting pissed

File: tgstation.dme (hunk 1 above)
Comment (doshMobile) [reply tgstation.dme:2294]: huh, i made sure to pull from the newest build so i dont know how that slipped in there

**Pull request merged**